spec/unit/installer_spec.rb

```diff
@@ -595,34 +595,28 @@ def @installer.clean_pod_sources
           @analysis_result = Installer::Analyzer::AnalysisResult.new(Pod::Installer::Analyzer::SpecsState.new, {}, {},
                                                                      [], Pod::Installer::Analyzer::SpecsState.new, [], [],
                                                                      Installer::Analyzer::PodfileDependencyCache.from_podfile(@installer.podfile))
-          @consumer = stub(:header_dir => 'myDir')
-          @spec = stub(:name => 'Spec', :test_specification? => false, :library_specification? => true, :non_library_specification? => false,
-                       :app_specification? => false, :consumer => @consumer)
-          @spec.stubs(:root => @spec)
-          @spec.stubs(:spec_type).returns(:library)
-          @spec.stubs(:module_name => 'Spec')
+          @spec = fixture_spec('banana-lib/BananaLib.podspec')
           @pod_targets = [PodTarget.new(config.sandbox, BuildType.static_library, {}, [], Platform.ios, [@spec],
-                                        [fixture_target_definition], nil)]
+                                        [fixture_target_definition], [], nil, '5.0')]
           @installer.stubs(:analysis_result).returns(@analysis_result)
           @installer.stubs(:pod_targets).returns(@pod_targets)
           @installer.stubs(:aggregate_targets).returns([])
         end
 
         it 'cleans the header stores' do
-          FileUtils.mkdir_p(config.sandbox.target_support_files_root)
```

Comment:
we now do `install!` below so this is not needed.

Verified test fails without my changes and passes with them.